friendship-quiz.js

```diff
@@ -1,3 +1,5 @@
+//editing for pull request
+
 // questions to be randomly asked on page load
 var questionsList = ["What are your/your friend's favorite foods?", "What are your/your friend's favorite animals?"];
 
```

Comment:
Some notes:
- using `console.log()` is a **great** idea while you're programming, but those lines aren't necessary for your program to function. Generally, the user of your program isn't interacting with the conwole, so those lines can be deleted when you're cleaning up your code!